src/Components/Publishing/Article.tsx

```diff
@@ -107,20 +106,15 @@ export class Article extends React.Component<ArticleProps> {
     return (
       <MediaContextProvider>
         <Theme>
-          <GridThemeProvider gridTheme={themeProps.grid}>
```

Comment:
For ref this was moved into https://github.com/artsy/palette/blob/master/src/Theme.tsx#L330